app/templates/components/forms/user-profile-form.hbs

```diff
@@ -1,8 +1,8 @@
-<form class="ui form" autocomplete="off" {{action 'submit' on='submit'}}>
+<form class="ui form {{if isLoading 'loading'}}" autocomplete="off" {{action 'submit' on='submit'}}>
   <div class="ui stackable grid">
     <div class="four wide column">
       {{widgets/forms/image-upload
```

Comment:
In the component `components/widgets/forms/image-upload.js` ... Line 28. Instead of the `later` function that now simulates an image upload